unicodetools/src/main/resources/org/unicode/uax42/output/tr42.html

```diff
@@ -49,18 +49,18 @@ <h1 style="text-align:center">Unicode Character Database in XML</h1>
                </tr>
                <tr>
                   <td valign="top">Date</td>
-                  <td valign="top" class="changed">2025-04-24</td>
+                  <td valign="top" class="changed">2025-07-08</td>
                </tr>
                <tr>
                   <td valign="top">This Version</td>
                   <td valign="top" class="changed">
-                     <a href="https://www.unicode.org/reports/tr42/tr42-37.html">https://www.unicode.org/reports/tr42/tr42-37.html</a>
+                     <a href="https://www.unicode.org/reports/tr42/tr42-38.html">https://www.unicode.org/reports/tr42/tr42-38.html</a>
```

Comment:
This Version is still 37.
The cleaned-up, de-yellowed version published with Unicode 17 will be 38.

On the server, we will go from tr37-37d1.html to tr37-37d2.html, together with your "draft 2" change above.